File: go/chapter8/album.go
package main

import "fmt"

func main() {
	fmt.Println(makeAlbum("Don Moen", "Thank you Lord", 1))
	fmt.Println(makeAlbum("Adele", "Go easy on me", 30))
	fmt.Println(makeAlbum("Jon Bellion", "I feel it", 1))
}

func makeAlbum(artist, title string, songs int) map[string]interface{} {
	album := map[string]interface{}{
		"artist": artist,
		"title":  title,
		"songs":  songs,
	}
	return album
}

File: python/chapter8/album.py
def make_album(artist: str, title: str, songs: int):
    album = {"artist": artist, "title": title, "songs": songs}
    return album


print(make_album("Don Moen", "Thank you Lord", 1))
print(make_album("Adele", "Go easy on me", 30))
print(make_album("Jon Bellion", "I feel it", 1))
